openapi/geonames/Module.md

```diff
@@ -0,0 +1,36 @@
+## Overview
+
+This is a generated connector for [Geonames API](https://www.geonames.org/export/JSON-webservices.html) OpenAPI specification.
+The Geonames API allows you to retrieve information about geo locations. . 
+
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* Create [Geonames Account](https://www.geonames.org/export/JSON-webservices.html)
+* Obtaining tokens by following [this link](https://www.geonames.org/export/web-services.html) 
+
+## Quickstart
+
+To use the Geonames connector in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import connector
+Import the ballerinax/geonames module into the Ballerina project.
+```ballerina
+import ballerinax/geonames;
+```
+### Step 2: Create a new connector instance
+You can use your organization name for authentication:
+```ballerina
+geonames:ApiKeysConfig apiKeyConfig = {
+    auth: {
+        username: <USERNAME>
+    }
+};
```

Comment:
```suggestion
geonames:ApiKeysConfig apiKeyConfig = {
     username: <USERNAME>
};
```

There is no field named `auth` inside `ApiKeysConfig`

Reply:
Fixed in https://github.com/ballerina-platform/openapi-connectors/commit/45c0ec66caf0d3ead895a90f7f2ff2ef6c431654